CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration.uc

```diff
@@ -490,6 +491,41 @@ static protected function EventListenerReturn AllowActionToSpawnRandomly (Object
 	return ELR_NoInterrupt;
 }
 
+static protected function EventListenerReturn MultiplyLootCaches(Object EventData, Object EventSource, XComGameState GameState, Name Event, Object CallbackData)
+{
+	local XComGameState_HeadquartersXCom XComHQ;
+	local XComGameState_MissionSite MissionState;
+	local XComGameState_Activity ActivityState;
+	local XComGameState_ActivityChain ChainState;
+	local XComGameState_Item ItemState;
+	local XComLWTuple Tuple;
+	
+	Tuple = XComLWTuple(EventData);
+	if (Tuple == none || Tuple.Id != 'MultiplyLootCaches') return ELR_NoInterrupt;
+
+	XComHQ = XComGameState_HeadquartersXCom(EventSource);
+	MissionState = XComGameState_MissionSite(`XCOMHISTORY.GetGameStateForObjectID(XComHQ.MissionRef.ObjectID));
+	ActivityState = class'XComGameState_Activity'.static.GetActivityFromPrimaryObject(MissionState);
+	ChainState = ActivityState.GetActivityChain();
+
+	if (ChainState.HasComplication('Complication_RewardInterception'))
+	{
+		ItemState = XComGameState_Item(Tuple.Data[1].o);
+		if (class'X2StrategyElement_DefaultComplications'.static.IsInterceptableItem(ItemState.GetMyTemplate()))
+		{
+			Tuple.Data[0].f = 0.5;
+		}
+		else
+		{
+			return ELR_NoInterrupt;
+		}
+	}
+
+	ChainState.ChainObjectRefs.AddItem(ItemState.GetReference());
+	ChainState.TriggerComplication(GameState, 'Complication_RewardInterception');
```

Comment:
You will call `TriggerComplication` once for each awarded item... I don't think that's what you want.

Also, `ChainObjectRefs` isn't really the place to pass the items - the array shouldn't be modified after the clain has been spawned/initialized